extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -1746,6 +1749,41 @@ describe('amp-a4a', () => {
       });
     });
 
+    it('should wait for first visible', () => {
+      let firstVisibleResolve;
+      viewerWhenVisibleMock.returns(
+        new Promise(resolve => {
+          firstVisibleResolve = resolve;
+        }));
+      expect(win.ampA4aValidationKeys).not.to.exist;
+      // Key fetch happens on A4A class construction.
+      const unusedA4a = new MockA4AImpl(a4aElement);  // eslint-disable-line no-unused-vars
+      const result = win.ampA4aValidationKeys;
+      return timerFor(win).promise(1).then(() => {
```

Comment:
I still don't understand why there's a 1-millisecond timer here. Can you explain?